physics/noahmp_tables.f90

```diff
@@ -510,11 +510,11 @@ module noahmp_tables
 
 !
     real :: cwpvt_table(mvt)       !< empirical canopy wind parameter
-      data ( cwpvt_table (i),i=1,mvt) / 0.18, 0.67, 0.18, 0.67, 0.29, 1.00,  &
-     &                            2.00, 1.30, 1.00, 5.00, 1.17, 1.67,        &
-     &                            1.67, 1.67, 0.18, 0.18, 0.18, 0.67,        &
-     &                            1.00, 0.18, 0.00, 0.00, 0.00, 0.00,        &
-     &                            0.00, 0.00, 0.00, 0.00, 0.00, 0.00 /
+      data ( cwpvt_table (i),i=1,mvt) / 0.09, 0.335, 0.09, 0.335, 0.145, 0.50, &
+     &                            1.00, 0.65,  0.50, 2.50, 0.585, 0.835,       &
+     &                            0.835, 0.835, 0.09, 0.09, 0.09, 0.335,       &
+     &                            0.835, 0.835, 0.09, 0.09, 0.09, 0.335,       &
```

Comment:
should be 0.5, 0.09, 0.00, 0.00, 0.00, 0.00?

Reply:
line 516

Reply:
Yes,  Thank you!